crates/matrix-sdk-base/src/latest_event.rs

```diff
@@ -75,20 +75,26 @@ pub struct LatestEvent {
     /// The member profile of the event' sender.
     #[serde(skip_serializing_if = "Option::is_none")]
     sender_profile: Option<MinimalRoomMemberEvent>,
+
+    // sender_room_member_event: Option<RoomMemberEvent
```

Comment:
nit: remove?

Reply:
Temp comment in a particular commit, but it was uncommented after.

Reply:
No it's not :stuck_out_tongue:

Reply:
Oh damned, it is now!